model_api/python/openvino/model_api/models/utils.py

```diff
@@ -22,6 +24,17 @@
 import numpy as np
 
 
+class AnomalyResult(NamedTuple):
+    """Results for anomaly models."""
+
+    anomaly_map: np.ndarray | None
+    box_labels: np.ndarray | None
+    pred_boxes: np.ndarray | None
```

Comment:
Is there any specific reason why we do not assign these to `None`

Reply:
Yeah they should be None as well